tutorials/multimodal_dapt_curation/ingest/main.py

```diff
@@ -0,0 +1,246 @@
+import argparse
+import json
+import os
+import re
+import shutil
+import time
+from base64 import b64decode
+from collections import defaultdict
+from io import BytesIO
+
+import arxiv as arxiv
+import pandas as pd
+from nv_ingest_client.client import Ingestor
+from PIL import Image
+
+# Constants for configuration and paths
+HTTP_HOST = os.environ.get("HTTP_HOST", "localhost")
+HTTP_PORT = os.environ.get("HTTP_PORT", "7670")
+TASK_QUEUE = os.environ.get("TASK_QUEUE", "morpheus_task_queue")
+
+MINIO_ACCESS_KEY = os.environ.get("MINIO_ACCESS_KEY", "minioadmin")
+MINIO_SECRET_KEY = os.environ.get("MINIO_SECRET_KEY", "minioadmin")
```

Comment:
```suggestion
```
The minio variables aren't necessary as there isn't any `store` task in the ingest job. I think a couple of these other variables can be removed too.